drupal/settings/all.settings.php

```diff
@@ -103,6 +103,11 @@ public function __toString() {
 // Ensure that administrators do not block drush access through the UI.
 $config['shield.settings']['allow_cli'] = TRUE;
 
+// Enforce correct solr server configuration (GOVCMS-4634)
+// Fix for 8.5.0 and the solr upgrade.
+$config['search_api.server.lagoon_solr']['backend_config']['connector_config']['path'] = '/';
```

Comment:
question: are we sure this is not going to prevent the bootstrap before the update hook ran (sorry, I dunno is solr module loads classes based on config values).